docs/superpowers/plans/2026-04-09-v4-to-v5-tls-defaulter.md

```
@@ -0,0 +1,416 @@
+git commit -m "Add cli image reference for v4-to-v5 TLS security profile defaulter Job"
+```
+
+---
+
+### Task 7: Verify all manifests
+
+- [ ] **Step 1: Verify all new files exist**
+
+Run: `ls -1 manifests/0000_05_kube-apiserver-operator_*`
+Expected:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)